dashboards-cloudfront.tf

```diff
@@ -2,26 +2,26 @@ variable "teamscloudfront1" {
   description = "map"
   default = {
     "team-a" = {
-      owneremail = "team-a@company.org"
-      teamname   = "Kiwi - di-bav-cri"
+      owneremail = "viveak.vadivelkarasan@digital.cabinet-office.gov.uk"
```

Comment:
Do you have a team email address you can use?  What do you use in the owner field in your CloudFormation tags?

Reply:
Team email created and updated here